apps/studio/src/server/modules/site/site.service.ts

```diff
@@ -213,3 +217,109 @@ export const clearSiteNotification = async ({
 
   return newSite
 }
+
+interface CreateSiteProps {
+  siteName: string
+}
+export const createSite = async ({ siteName }: CreateSiteProps) => {
+  const siteId = await db.transaction().execute(async (tx) => {
+    const { id: siteId } = await tx
+      .insertInto("Site")
+      .values({
+        name: siteName,
+        theme: jsonb({
+          colors: {
+            brand: {
+              canvas: {
+                alt: "#bfcfd7",
+                default: "#e6ecef",
+                inverse: "#00405f",
+                backdrop: "#80a0af",
+              },
+              interaction: {
+                hover: "#002e44",
+                default: "#00405f",
+                pressed: "#00283b",
+              },
+            },
+          },
+        }),
+        config: jsonb({
+          theme: "isomer-next",
+          siteName,
+          logoUrl: "https://www.isomer.gov.sg/images/isomer-logo.svg",
+          search: undefined,
+          isGovernment: true,
+        }),
+      })
+      .onConflict((oc) =>
+        oc
+          .column("name")
+          .doUpdateSet((eb) => ({ name: eb.ref("excluded.name") })),
+      )
+      .returning("id")
+      .executeTakeFirstOrThrow()
+
+    await tx
+      .insertInto("Footer")
+      .values({
+        siteId,
+        content: jsonb(FOOTER),
+      })
+      .onConflict((oc) =>
+        oc
+          .column("siteId")
+          .doUpdateSet((eb) => ({ siteId: eb.ref("excluded.siteId") })),
+      )
+      .execute()
+
+    await tx
+      .insertInto("Navbar")
+      .values({
+        siteId,
+        content: jsonb(NAV_BAR_ITEMS),
+      })
+      .onConflict((oc) =>
+        oc
+          .column("siteId")
+          .doUpdateSet((eb) => ({ siteId: eb.ref("excluded.siteId") })),
+      )
+      .execute()
+
+    const { id: blobId } = await tx
+      .insertInto("Blob")
+      .values({ content: jsonb(PAGE_BLOB) })
+      .returning("id")
+      .executeTakeFirstOrThrow()
+
+    await tx
+      .insertInto("Resource")
+      .values({
+        draftBlobId: String(blobId),
+        permalink: "",
+        siteId,
+        type: "RootPage",
+        title: "Home",
+      })
+
+      .onConflict((oc) =>
+        oc.column("draftBlobId").doUpdateSet((eb) => ({
+          draftBlobId: eb.ref("excluded.draftBlobId"),
+        })),
+      )
+      .executeTakeFirstOrThrow()
+
+    console.log(`Added site ${siteName} with id ${siteId} to database`)
```

Comment:
nit: now don't need this already since it got shifted into bacskend

Reply:
nice catch! copied the script over directly and didn't remove - fixed in https://github.com/opengovsg/isomer/pull/1241/commits/9d644e0e5666ab89c4eb441d59adafebe844e762